catowseriOS/catowser/BrowserContent/TopSites/TitledImageView.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  TitledImageView.swift
+//  catowser
+//
+//  Created by Andrey Ermoshin on 24.04.2023.
+//  Copyright © 2023 andreiermoshin. All rights reserved.
+//
+
+import SwiftUI
+import CottonCoreBaseKit
+
+struct TitledImageView: View {
+    private let site: Site
```

Comment:
maybe need to use a Swift protocol to only have 3 needed properties/functions from `Site` type and not any other properties